pom.xml

```diff
@@ -152,7 +151,7 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-all</artifactId> <!-- Use 'netty-all' for 4.0 or above -->
-            <version>4.1.27.Final</version>
+            <version>4.1.33.Final</version>
```

Comment:
There is a netty on nsx side. Updating netty always a pretty hard process. We should figure out if we can do that

Reply:
4.1.33 is the netty version being used in nsx.

Reply:
ok